### DISCUSSION TITLE
Clarify allowed values of version field

### DISCUSSION THREAD

**Discussion by ryan-thomas-peterson** [CMSgov/price-transparency-guide#762] [Q&A]
The version format in the examples that are present specify a version of type 1.1.2 however, the MRF validator tool will use the version defined in the file if and only if the version is of format v1.1.2 (with a leading lower case v).  I understand the validator tool can specify the specific version explicitly.  Is there a preferred version definition?  Is either variant compliant?

**Comment by sampathsurekha02** [comment]
@ryan-thomas-peterson - Did you receive a resolution for this issue?

**Reply by ryan-thomas-peterson** [reply]
I have not, no.
